packages/assets-controllers/CHANGELOG.md

```diff
@@ -5,19 +5,52 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+    - We retry each request up to 3 times using a randomized exponential backoff strategy
+    - If the token price update still fails 12 times consecutively (3 update attempts, each of which has 4 calls due to retries), we stop trying for 30 minutes before we try again.
+- Add polling by `networkClientId` to `AccountTrackerController` ([#3586](https://github.com/MetaMask/core/pull/3586))
+  - A new state property, `accountByChainId` has been added for keeping track of account balances across chains
+  - `AccountTrackerController` implements `PollingController` and can now poll by `networkClientId` via the new methods `startPollingByNetworkClientId`, `stopPollingByPollingToken`, and `stopPollingByPollingToken`.
+  - `AccountTrackerController` accepts an optional `networkClientId` value on the `refresh` method
+  - `AccountTrackerController` accepts an optional `networkClientId` value as the last parameter of the `syncBalanceWithAddresses ` method
+- Support token detection on Base and zkSync ([#3584](https://github.com/MetaMask/core/pull/3584))
+- Support token detection on Arbitrum and Optimism ([#2035](https://github.com/MetaMask/core/pull/2035))
 
 ### Changed
-- **BREAKING:** TokenRatesController now takes a required argument `tokenPricesService` ([#3600](https://github.com/MetaMask/core/pull/3600))
+- **BREAKING:** `TokenRatesController` now takes a required argument `tokenPricesService` ([#3600](https://github.com/MetaMask/core/pull/3600))
   - This object is responsible for fetching the prices for tokens held by this controller.
 - **BREAKING:** Update signature of `TokenRatesController.updateExchangeRatesByChainId` ([#3600](https://github.com/MetaMask/core/pull/3600), [#3653](https://github.com/MetaMask/core/pull/3653))
   - Change the type of `tokenAddresses` from `string[]` to `Hex[]`
-- **BREAKING:** Change signature of `TokenRatesController.fetchAndMapExchangeRates` ([#3600](https://github.com/MetaMask/core/pull/3600), [#3653](https://github.com/MetaMask/core/pull/3653))
-  - This method now takes an object with shape `{ tokenAddresses: Hex[]; chainId: Hex; nativeCurrency: string; }` rather than positional arguments
-- Update TokenListController to fetch prefiltered set of tokens from the API, reducing response data and removing the need for filtering logic ([#2054](https://github.com/MetaMask/core/pull/2054))
-- Update TokenRatesController to request token rates from the Price API in batches of 100 ([#3650](https://github.com/MetaMask/core/pull/3650))
+- **BREAKING:** `AccountTrackerController` constructor params object requires `getCurrentChainId` and `getNetworkClientById` hooks ([#3586](https://github.com/MetaMask/core/pull/3586))
+  - These are needed for the new "polling by `networkClientId`" feature
+- **BREAKING**: `AccountTrackerController` adds a mutex to `refresh` making it only possible for one call to be executed at time ([#3586](https://github.com/MetaMask/core/pull/3586))
```

Comment:
Hmm. Good question. It is a notable difference in behavior that isn't strictly compatible with the previous behavior, but I'm having trouble thinking of a scenario where it would be "breaking".

I guess you could now introduce deadlocks that weren't possible previously, but only by using this method in a very strange way :sweat_smile:

Reply:
Haha sorry about that. Feel free to remove the BREAKING label if you feel that it isn't accurate.

Reply:
I can remove the "BREAKING" if someone finds that preferable or finds this confusing, but otherwise I'll leave it. I think there's a reasonable case to be made either way, definitely a gray area.

Reply:
Okay, better to be safe than sorry.